docs/writing-stories/loaders.md

```diff
@@ -2,30 +2,36 @@
 
-Loaders can be used to load any asset, typically as a performance optimization. They were designed for to lazy load components and other large story imports. They can also be used to load remote API data to be used in a story. However, [Args](./args.md) is the recommended way to manage story data, and we're building up an ecosystem of tools and techniques around Args which might not be compatible with loaded data.
+Loaders can be used to load any asset, lazy load components, or fetch data from a remote API. This feature was designed as a performance optimization to handle large story imports. However, [Args](./args.md) is the recommended way to manage story data. We're building up an ecosystem of tools and techniques around Args that might not be compatible with loaded data.
 
-Loaders are an advanced feature ("escape hatch") and we only recommend using them if you have a specific need that can't be fulfilled by other means. They are experimental in Storybook 6.1 and the APIs are subject to change outside of the normal semver cycle.
+They are an advanced feature (i.e., escape hatch), and we only recommend using them if you have a specific need that other means can't fulfill. They are experimental in Storybook 6.1, and the APIs are subject to change outside of the normal semver cycle.
 
 ## Fetching API data
 
-Stories are isolated component examples that render internal data that's defined as part of the story or alongside the story as [args](./args.md).
+Stories are isolated component examples that render internal data defined as part of the story or alongside the story as [args](./args.md).
 
-Loaders are useful when you need to load story data externally, e.g. from a remote API. Consider the following example that fetches a todo item for display in a todo list:
+Loaders are helpful when you need to load story data externally (e.g., from a remote API). Consider the following example that fetches a todo item to display in a todo list:
 
 <!-- prettier-ignore-start -->
 
 <CodeSnippets
   paths={[
     'react/loader-story.js.mdx',
+    'react/loader-story.mdx.mdx',
+    'vue/loader-story.3.js.mdx',
+    'vue/loader-story.mdx.mdx',
+    'angular/loader-story.ts.mdx',
+    'angular/loader-story.mdx.mdx',
     'svelte/loader-story.js.mdx',
+    'svelte/loader-story.mdx.mdx',
   ]}
 />
 
 <!-- prettier-ignore-end -->
 
-The loaded data is combined into a `loaded` field on the story context, which is the second argument to a story function. In this example we spread the story's args in first, so they take priority over the static data provided by the loader.
+The response obtained from the remote API call is combined into a `loaded` field on the story context, which is the second argument to a story function. For example, in React, the story's args were spread first to prioritize them over the static data provided by the loader. Other frameworks will not need such requisite.
```

Comment:
> frameworks will not need such requisite.

not sure what this means

Reply:
I'll clarify it. For instance with Angular we're not spreading args at all. This is only applicable to React.

Reply:
With the latest commit, I've addressed the typos (thanks for this @shilman ), and added a bit of context on this item of feedback. Let me know if you're ok and we can merge it.